packages/phone-number-privacy/test/salt-generation/bls-signature.test.ts

```diff
@@ -1,11 +1,30 @@
 import threshold from 'blind-threshold-bls'
-import { computeBlindedSignature } from '../../src/bls/bls-signature'
+import { BLSCryptographyClient } from '../../src/bls/bls-cryptography-client'
+import config from '../../src/config'
+
+const USING_MOCK = config.keyVault.azureClientSecret === 'useMock'
 
 describe(`BLS service computes signature`, () => {
+  beforeEach(() => {
+    // Use mock client if env vars not specified
+    if (!USING_MOCK) {
```

Comment:
is it not expected that the test will always run with the DEV environment configs?  seems we could just expect that we are using the mock, no?

Reply:
Correct, it will always use mocks as-is. The idea is that when a dev wants to locally test the an actual KV instance, they would add the keyvault variables to the config file. Devs making changes that affect this logic would be expected to test out the non-mocked scenario to ensure it all works. I'll add this to our dev readme to make it more clear.